taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -51,3 +51,4 @@
 @import url("_imports/trumps/s-footer.css");
 @import url("_imports/trumps/s-blockquote.css");
 @import url("_imports/trumps/u-empty.css");
+@import url("@tacc/core-styles/src/lib/_imports/trumps/u-mailto-text-replace.css");
```

Comment:
This lets a CMS editor use the `u-mailto-text-replace` utility class on any page. The utility class comes from https://github.com/TACC/Core-Styles/pull/44.

Reply:
_P.S. Many other stylesheets here actually come from @tacc/core-styles also, but I imported them before Core-Styles was available on NPM, so the build tool does some spelunking to figure out where to get stylesheets from. Magic. Bad. Which is why I start using `@tacc/core-styles` in import statements._